commands/FBDebugCommands.py

```diff
@@ -189,6 +191,58 @@ def run(self, arguments, options):
     fb.evaluateEffect('[[UIApplication sharedApplication] performSelector:@selector(_performMemoryWarning)]')
 
 
+def switchBreakpointState(expression,on):
+  ci = lldb.debugger.GetCommandInterpreter()
+  res = lldb.SBCommandReturnObject()
+
+  ci.HandleCommand('br list',res)
+  listStr = res.GetOutput()
+
+  br_list = listStr.splitlines()
+  expression_pattern = re.compile(r'{}'.format(expression),re.I)
+  id_pattern = re.compile(r'^\s*([1-9]\d*)(\.\d+)?',re.I)
+  for br in br_list:
+    if expression_pattern.search(br) and id_pattern.search(br):
+        id = id_pattern.search(br).group()
+        if id:
+          print br
+          if on:
+            lldb.debugger.HandleCommand('br enable {}'.format(id))
+          else:
+            lldb.debugger.HandleCommand('br disable {}'.format(id))
+
+class FBMethodBreakpointEnableCommand(fb.FBCommand):
+  def name(self):
+    return 'benable'
```

Comment:
lldb has an alias command named `rb` for **r**egex **b**reakpoint. I think it makes sense to call this `rbenable` and `rbdisable`, what do you think?